models/src/main/scala/it/pagopa/interop/tenantmanagement/model/persistence/events.scala

```diff
@@ -6,8 +6,10 @@ import java.util.UUID
 
 sealed trait Event extends Persistable with ProjectableEvent
 
-final case class TenantCreated(tenant: PersistentTenant)                    extends Event
-final case class TenantUpdated(tenant: PersistentTenant)                    extends Event
-final case class TenantDeleted(tenantId: String)                            extends Event
-final case class SelfcareMappingCreated(selfcareId: String, tenantId: UUID) extends Event
-final case class SelfcareMappingDeleted(selfcareId: String)                 extends Event
+final case class TenantCreated(tenant: PersistentTenant)                     extends Event
+final case class TenantUpdated(tenant: PersistentTenant)                     extends Event
+final case class TenantDeleted(tenantId: String)                             extends Event
+final case class SelfcareMappingCreated(selfcareId: String, tenantId: UUID)  extends Event
+final case class SelfcareMappingDeleted(selfcareId: String)                  extends Event
+final case class TenantMailAdded(tenantId: UUID, mail: PersistentTenantMail) extends Event
```

Comment:
```suggestion
final case class TenantMailAdded(tenantId: UUID, mailId:String, tenant: PersistentTenant) extends Event
```
@galales are you agree?

Reply:
yep I agree with the suggestion